app/views/estimate_flow/forms/_dependant_details.html.slim

```diff
@@ -1,3 +1,6 @@
+- dependant_allowance = FeatureFlags.enabled?(:el_809) ? "new" : "old"
```

Comment:
i would suggest using legacy instead of old, just ti be consistent with how we have handled these changes in the past. certainly not a blocker though if you want to keep it like this, as ultimately it will be removed

Reply:
I should have been more clear here, if we use legacy instead of old and dont enter anything for 'new' so it looks like this:
FeatureFlags.enabled?(:el_809) ? "" : "legacy"

This way we just need to remove the legacy translations at some point and then there is no need to change the translations to remove the "new" part. It just makes removing feature flag items easier.

Reply:
Although we'd need to be a little careful here, because a blank string here would mean the translation key ended up being "controlled_income_exclusion_" with a dangling underscore at the end, which is not ideal. 

You could do:

```
suffix = FeatureFlags.enabled?(:el_809) ? "" : "_legacy"
```

And then on the next line remove the underscore i.e.:

```
income_exclusion_key = "#{@estimate.level_of_help}_income_exclusion#{suffix}"
```